azure-devops/azext_devops/dev/boards/relations.py

```diff
@@ -21,36 +21,47 @@ def get_relation_types_show(organization=None, detect=None):
     return client.get_relation_types()
 
 
-def add_relation(id, relation_type, target_id, organization=None, detect=None):  # pylint: disable=redefined-builtin
+def add_relation(id, relation_type, target_id=None, target_url=None, organization=None, detect=None):  # pylint: disable=redefined-builtin
     """ Add relation(s) to work item.
     """
+
+    if target_id is None and target_url is None:
+        raise CLIError('--target-id or --target-url shall be provided')
```

Comment:
nit: should we say `must` instead of `shall`?

Reply:
Done